### ISSUE TITLE
Segault on `bun upgrade`

### ISSUE THREAD

**Issue by headquarter8302** [oven-sh/bun#11804]
### How can we reproduce the crash?

1. Run `bun upgrade` in any terminal or console environment

### Relevant log output

```shell
============================================================
Bun v1.1.11 (4fb6056f) Windows x64
Args: "C:\Users\ASUS\.bun\bin\bun.exe" "upgrade"
Elapsed: 785ms | User: 0ms | Sys: 15ms
RSS: 55.99MB | Peak: 55.99MB | Commit: 117.02MB | Faults: 13878

panic(thread 18712): Segmentation fault at address 0xFFFFFFFFFFFFFFFF
oh no: Bun has crashed. This indicates a bug in Bun, not your code.
```

### Stack Trace (bun.report)

Bun v1.1.11 ([`4fb6056`](<https://github.com/oven-sh/bun/tree/4fb6056f85c9cf643caa9c2afdefe4917c513fb5>)) on windows x86_64 [UpgradeCommand]

Segmentation fault at address 0xFFFFFFFFFFFFFFFF

- ??? at `0x1117b` in VCRUNTIME140.dll
- [`fixed_buffer_stream.zig:70`](<https://github.com/oven-sh/bun/blob/4fb6056f85c9cf643caa9c2afdefe4917c513fb5/src/deps/zig/lib/std/io/fixed_buffer_stream.zig#L70>): `write`
- [`writer.zig:23`](<https://github.com/oven-sh/bun/blob/4fb6056f85c9cf643caa9c2afdefe4917c513fb5/src/deps/zig/lib/std/io/writer.zig#L23>): `writeAll`
- [`http.zig:3191`](<https://github.com/oven-sh/bun/blob/4fb6056f85c9cf643caa9c2afdefe4917c513fb5/src/http.zig#L3191>): `handleResponseBodyFromSinglePacket`
- [`http.zig:3176`](<https://github.com/oven-sh/bun/blob/4fb6056f85c9cf643caa9c2afdefe4917c513fb5/src/http.zig#L3176>): `handleResponseBody`
- [`http.zig:2788`](<https://github.com/oven-sh/bun/blob/4fb6056f85c9cf643caa9c2afdefe4917c513fb5/src/http.zig#L2788>): `onData`
- [`uws.zig:916`](<https://github.com/oven-sh/bun/blob/4fb6056f85c9cf643caa9c2afdefe4917c513fb5/src/deps/uws.zig#L916>): `on_data`
- `??`
- `??`
- `uv__process_poll_req`

**Comment by headquarter8302** [comment]
I noticed that the error happened on VCRUNTIME140.dll, is it possible that I may have a corrupted dependency installed on my Windows machine?

**Comment by Jarred-Sumner** [comment]
This was fixed in Bun v1.1.12. Please upgrade via Powershell:

```powershell
powershell -c "irm bun.sh/install.ps1 | iex"
```
